app/models/user.rb

```diff
@@ -0,0 +1,9 @@
+class User < ApplicationRecord
+  has_many :posts, foreign_key: 'author_id', dependent: :destroy, counter_cache: true
+  has_many :comments, foreign_key: 'user_id', dependent: :destroy
+  has_many :likes, foreign_key: 'user_id', dependent: :destroy
```

Comment:
- [Optional] In the posts table you change user to author and to remind Rails of this changes you explicitly showing here that  `author_id` should be the foreign key in the post table instead of the default `user_id`. In the comments and likes tables, you didn't alter user to author hence there's no reason to explicitly point user_id as the foreign keys here. Rails is intelligent enough :smile: to figure out this. So you could just do something like:
```
has_many :comments
has_many :likes
```
 But I really think you're doing well in this project :+1: